index.js

```diff
@@ -0,0 +1,121 @@
+//DELIVERABLE 1
+//When page loads:
+//Diaplay player1 info- name, nickname, photo, likes, goals
+//Fetch player info 
+
+
+
+fetch('http://localhost:3000/players')
+    .then(resp => resp.json())
+    .then(player => renderPlayer(player[0]))
```

Comment:
Here, you're using the wrong URL for the fetch, which is why you aren't getting back the correct object (player with goals array). Make sure to read the Readme! The API requests are very important to get right.

```js
fetch('http://localhost:3000/players/1')
    .then(resp => resp.json())
    .then(firstPlayer => renderPlayer(firstPlayer))
```